packages/react-core/src/components/Popover/Popover.tsx

```diff
@@ -112,6 +112,12 @@ export interface PopoverProps {
   headerContent?: React.ReactNode | ((hide: () => void) => React.ReactNode);
   /** Sets the heading level to use for the popover header. Default is h6. */
   headerComponent?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  /** Composable header component. Use with the PopoverHeader, PopoverHeaderText, and PopoverHeaderIcon components. To manually close
+   * a popover after an action within the bodyContent you can provide a function which will receive a callback as an argument to hide the popover.
+   */
+  header?: React.ReactNode | ((hide: () => void) => React.ReactNode);
+  /** Variants for an alert popover */
+  alertVariant?: 'default' | 'info' | 'warning' | 'success' | 'danger';
```

Comment:
So, I think the alertVariant on its own does not make sense.  Setting this prop does not make the popover an alert popover.  I wonder if we can improve on the API?
@jschuler I would be interested in your thoughts.

Reply:
@tlabaj how about `alertLevelVariant` or `alertSeverityVariant`

Reply:
@nicole, those suggestions are good.  Maybe some more context in the description of what it does will also help.

Reply:
It would make more sense for me to find a prop like that on the `PopoverHeader` component, since it affects the icon and text color in there. But if we anticipate more changes to the entire Popover it would make sense to be here.

If the prop comment explains that it modifies the header icon and text colors, that would make it clearer what change it does.